2021-12-29 03:57:26,691 INFO    MainThread:2222 [wandb_setup.py:_flush():71] setting env: {}
2021-12-29 03:57:26,692 INFO    MainThread:2222 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-29 03:57:26,692 INFO    MainThread:2222 [wandb_init.py:_log_setup():371] Logging user logs to /content/drive/MyDrive/Colab Notebooks/stylegan2-pytorch/wandb/run-20211229_035726-cxeenjh9/logs/debug.log
2021-12-29 03:57:26,693 INFO    MainThread:2222 [wandb_init.py:_log_setup():372] Logging internal logs to /content/drive/MyDrive/Colab Notebooks/stylegan2-pytorch/wandb/run-20211229_035726-cxeenjh9/logs/debug-internal.log
2021-12-29 03:57:26,693 INFO    MainThread:2222 [wandb_init.py:init():404] calling init triggers
2021-12-29 03:57:26,693 INFO    MainThread:2222 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-12-29 03:57:26,693 INFO    MainThread:2222 [wandb_init.py:init():460] starting backend
2021-12-29 03:57:26,693 INFO    MainThread:2222 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-29 03:57:26,727 INFO    MainThread:2222 [backend.py:ensure_launched():216] starting backend process...
2021-12-29 03:57:26,771 INFO    MainThread:2222 [backend.py:ensure_launched():222] started backend process with pid: 2254
2021-12-29 03:57:26,780 INFO    MainThread:2222 [wandb_init.py:init():469] backend started and connected
2021-12-29 03:57:26,794 INFO    MainThread:2222 [wandb_init.py:init():533] updated telemetry
2021-12-29 03:57:26,928 INFO    MainThread:2222 [wandb_init.py:init():563] communicating current version
2021-12-29 03:57:27,749 INFO    MainThread:2222 [wandb_init.py:init():568] got version response 
2021-12-29 03:57:27,750 INFO    MainThread:2222 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2021-12-29 03:57:28,140 INFO    MainThread:2222 [wandb_init.py:init():601] run resumed
2021-12-29 03:57:28,145 INFO    MainThread:2222 [wandb_init.py:init():606] starting run threads in backend
2021-12-29 03:57:30,667 INFO    MainThread:2222 [wandb_run.py:_console_start():1810] atexit reg
2021-12-29 03:57:30,670 INFO    MainThread:2222 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2021-12-29 03:57:30,670 INFO    MainThread:2222 [wandb_run.py:_redirect():1689] Redirecting console.
2021-12-29 03:57:30,675 INFO    MainThread:2222 [wandb_run.py:_redirect():1745] Redirects installed.
2021-12-29 03:57:30,675 INFO    MainThread:2222 [wandb_init.py:init():633] run started, returning control to user process
2021-12-29 03:57:31,525 INFO    MainThread:2222 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2021-12-29 03:57:31,533 INFO    MainThread:2222 [wandb_run.py:_restore():1752] restore
2021-12-29 03:57:33,696 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
}
pusher_stats {
}

2021-12-29 03:57:33,800 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
}
pusher_stats {
}

2021-12-29 03:57:33,960 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
}
pusher_stats {
}

2021-12-29 03:57:34,172 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  total_bytes: 733
}

2021-12-29 03:57:34,274 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  total_bytes: 1397
}

2021-12-29 03:57:34,376 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  total_bytes: 1397
}

2021-12-29 03:57:34,478 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  total_bytes: 1397
}

2021-12-29 03:57:34,783 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  total_bytes: 1397
}

2021-12-29 03:57:35,151 INFO    MainThread:2222 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 3
}
pusher_stats {
  total_bytes: 1397
}
local_info {
}

2021-12-29 03:57:36,618 INFO    MainThread:2222 [wandb_run.py:_append_summary():2085] rendering summary
2021-12-29 03:57:36,618 INFO    MainThread:2222 [wandb_run.py:_append_files():2180] logging synced files
